crud.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any
+
+from sqlalchemy.orm import Session, Query
+
+from db import models
+import schemas
+
+
+def get_all_authors(db: Session) -> Query[Any]:
```

Comment:
Why do you use Any? It is bad practice!

Reply:
I got it. I should use models class instead Any. Am I right?

Reply:
Yes, you are right